lib/middleman-hashicorp/extension.rb

```diff
@@ -100,7 +117,113 @@ def initialize(app, options_hash = {}, &block)
     end
   end
 
+  expose_to_template :segmentId
+
+  # If a production Segment ID is provided, use it, otherwise use staging by default
+  def segmentId()
+    options.segment_production_key
+  end
```

Comment:
Ah so we ended up moving the segment key into the JS config post-GDPR, so I think we can take this out now. Sorry for the backtrack!